cookbooks/zookeeper/README.md

```diff
@@ -15,23 +15,6 @@ bundle exec berks vendor cookbooks
 2. Commit and updated `cookbooks` directory
 3. Tag the commit that we want to release with format `<APP-VERSION>-<REVISION>`
 
-## Building Image
```

Comment:
why did you remove this README section?

Reply:
I do not, but somehow it removed. I will revert it back.